truss/remote/baseten/remote.py

```diff
@@ -57,3 +61,43 @@ def push(self, truss_handle: TrussHandle, model_name: str):  # type: ignore
             service_url=f"{self._remote_url}/model_versions/{model_version_id}",
             truss_handle=truss_handle,
         )
+
+    def watch(self, watch_path: Path, logger: logging.Logger):
```

Comment:
this seems like it should be called patch, not watch, right? Not sure where you're wathcing here.